javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,132 @@
+
+//Fllight booking fullname function
+function gettFullname (firstname, surname, useFormalName, gender) {
+    if (useFormalName == true && gender == 'male') {
+        return `Lord ${firstname} ${surname}`
+    } else if (useFormalName == true && gender == 'female'){
+        return `Mis ${firstname} ${surname}`
+    }else {
+        return `${firstname} ${surname}`
+    }
+}
+
+let fullname1 = gettFullname('Leonardo', 'DiCaprio', true, 'male')
+let fullname2 = gettFullname('Angellina', 'Jolie', true, 'female')
+let fullname = gettFullname('Joey', 'Ttibbiani', false, 'male')
+
```

Comment:
nice! :) you can also console.log your function so you can see your returns. 
So you can simply add a console.log(fullname1)